defaults/main.yaml

```diff
@@ -83,9 +83,8 @@ pulp_server_config__default:
 pulp_server_config__custom: {}
 
 # Final Pulp Server config
-pulp_server_config: "{{
-  pulp_server_config__default.update(pulp_server_config__custom) }}{{
-  pulp_server_config__default }}"
+pulp_server_config: "{{ pulp_server_config__default |
```

Comment:
Please could you format it like this?

```
pulp_server_config: "{{
  pulp_server_config__default |
  combine(pulp_server_config__custom) }}
```

The same in all other cases bellow.